submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/application_wide/template.md

```diff
@@ -0,0 +1,44 @@
+# Cross-Site Request Forgery (Application-Wide)
+
+## Overview of the Vulnerability
+
+Cross-Site Request Forgery (CSRF) occurs when requests to the application are submitted on behalf of an authenticated user without their knowledge via crafted, malicious code which can be in the form of a link the user clicks. The application is unable to distinguish between the malicious request and a legitimate request from the user.
+
+Application-wide CSRF is possible for this application, allowing an attacker to submit requests to the application on behalf of an authenticated user on multiple endpoints. Additionally, the attacker needs to socially engineer the user to click on a link, or paste the malicious code into the user’s browser. If successful, the code will execute within that user’s browser in the context of this domain.
```

Comment:
We want to highlight why the bugs are P2 bugs, we want to explain Why the issue is so impactful of a security issue and why we want to manage it, focus on Application Wide and the potential impact for users.